source/common/stats/stat_merger.h

```diff
@@ -42,12 +42,43 @@ class StatMerger {
 
   StatMerger(Stats::Store& target_store);
 
-  // Merge the values of stats_proto into stats_store. Counters are always straightforward
-  // addition, while gauges default to addition but have exceptions.
+  /**
+   * Merge the values of stats_proto into stats_store. Counters are always
+   * straightforward addition, while gauges default to addition but have
+   * exceptions.
+   *
+   * @param counter_deltas map of counter changes from parent
+   * @param counter_deltas map of gauge changes from parent
+   * @param dynamics information about which segments of the names are dynamic.
+   */
   void mergeStats(const Protobuf::Map<std::string, uint64_t>& counter_deltas,
                   const Protobuf::Map<std::string, uint64_t>& gauges,
```

Comment:
This is pre-existing code, but uint64_t here is a bit odd. For subtraction does the rely on wrapping? Can you maybe update the comments to make this more clear? Same elsewhere where this applies.

Reply:
Sorry nevermind this is not a delta, I see.